### PR TITLE
Arbitrate queued activation requests

### PR DESCRIPTION
**Pull request by bzwei**
If multiple requests have been queued for an activation we don't have to execute them one by one. The arbitrate logic consolidates them base on the sequence only, regardless of the current status of the activation. Many of the consolidations may not actually happen because the view already has validation based on the current status. Furthermore the manager has validations too before the request is actually executed.

### CODE REVIEW

File: tests/integration/tasks/test_activation_request_queue.py [hunk 1]
```diff
@@ -42,19 +42,79 @@ def init_activations():
...
+        {
+            "queued": [
+                ActivationRequest.RESTART,
+                ActivationRequest.STOP,
+                ActivationRequest.START,
+            ],
+            "dequeued": [
+                ActivationRequest.STOP,
+                ActivationRequest.START,
+            ],
```
Comment (Alex-Izquierdo) [inline tests/integration/tasks/test_activation_request_queue.py:98]: I think in this case should be contrary. Restart is stop+start, so the dequeued output should be restart, why schedule a stop and later a start when we can do both in the same verb manager.restart()

File: src/aap_eda/tasks/activation_request_queue.py [hunk 2]
```diff
@@ -39,3 +40,50 @@ def list_activations() -> list[int]:
...
+def _arbitrate(
+    requests: list[ActivationRequestQueue],
+) -> list[ActivationRequestQueue]:
+    if len(requests) < 2:
+        return requests
+
+    ref_request = None
+    qualified_requests = []
+    starts = [ActivationRequest.START, ActivationRequest.RESTART]
+    for request in requests:
```
Comment (Alex-Izquierdo) [inline src/aap_eda/tasks/activation_request_queue.py:54]: Suggestion:

File: src/aap_eda/tasks/activation_request_queue.py [hunk 3]
```diff
@@ -23,9 +23,10 @@ def push(activation_id: int, request: ActivationRequest) -> None:
 
 
 def peek_all(activation_id: int) -> list[ActivationRequestQueue]:
-    return ActivationRequestQueue.objects.filter(
+    requests = ActivationRequestQueue.objects.filter(
         activation_id=activation_id
     ).all()
+    return _arbitrate(requests)
```
[the remark this answers is not in the document]
Comment (bzwei) [reply src/aap_eda/tasks/activation_request_queue.py:29]: Per our discussion, at this moment we don't need a very fancy arbitrator. We can refactor it if we really have such need.

File: tests/integration/tasks/test_activation_request_queue.py (hunk 1 above)
Comment (bzwei) [reply tests/integration/tasks/test_activation_request_queue.py:98]: Per our discussion there will no RESTART, STOP, START combination in real.

**Pull request merged**